contracts/usul/LinearTokenVoting.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import "./BaseTokenVoting.sol";
+import "./BaseQuorumPercent.sol";
+
+/// @title A Usul strategy that enables linear token voting
+contract LinearTokenVoting is BaseTokenVoting, BaseQuorumPercent {
+    ERC20Votes public governanceToken;
+
+    constructor(
+        address _owner,
+        ERC20Votes _governanceToken,
+        address _usulModule,
+        uint256 _votingPeriod,
+        uint256 quorumNumerator_,
```

Comment:
why is this underscore at the end here?

Reply:
as well as `name_` below

Reply:
Good call, yeah this was leftover from Usul. Just pushed up fix for this